### PR TITLE
animation div sticky correction

### LINKED ISSUE #5985: Animation vanishes on scrolling
### Description

In [this](https://layer5.io/cloud-native-management/meshery/operating-cloud-native-infra) page, when we scroll through the page, the animation shows up for a second and then vanishes away. 

### Expected Behavior

We want the animation to load properly and not vanish away, when we scroll down the page.

### Screenshots

https://github.com/user-attachments/assets/ef567bd3-e7a0-414a-91d7-e83f4e1e638c

### Environment:
- Host OS: MacOs
- Browser: Google Chrome, Safari

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by balagowda**
**Description**

This PR fixes #5985

https://github.com/user-attachments/assets/a0d4c5a1-3e4c-4896-9946-5f69c61f56db

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 9d62e6b0c61807aa96f7000886f01394415b2cca at: https://67694a5916d269264ab11c3b--layer5.netlify.app

File: src/sections/General/Navigation/navigation.style.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import styled from "styled-components";
 
 const NavigationWrap = styled.header`
-  position: sticky;
+  // position: sticky;
```
Comment (sudhanshutech) [inline src/sections/General/Navigation/navigation.style.js:4]: remove it if not needed, don't keep it as comment

File: src/sections/Meshery/How-meshery-works/howitworks.style.js [hunk 2]
```diff
@@ -71,6 +71,7 @@ h2 {
   }
   
   .diagram {
+    margin-bottom:150px;
```
Comment (sudhanshutech) [inline src/sections/Meshery/How-meshery-works/howitworks.style.js:74]: why extra margin?

File: src/sections/app.style.js [hunk 3]
```diff
@@ -64,7 +64,7 @@ body {
   transition-timing-function: cubic-bezier(0.2, 0.8, 0.2, 1);
 
   font-family: "Qanelas Soft", "Open Sans", sans-serif;
-  overflow-x: hidden !important;
+  // overflow-x: hidden !important;
```
Comment (sudhanshutech) [inline src/sections/app.style.js:67]: same here remove it

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
@balagowda incorporate the feedback and rest looks good.✔️

File: src/sections/Meshery/How-meshery-works/howitworks.style.js (hunk 2 above)
Comment (balagowda) [reply src/sections/Meshery/How-meshery-works/howitworks.style.js:74]: @sudhanshutech Left div will go to the end of the parent div, compared to the right div, which looks bad. So, I added a margin there.

Comment (balagowda) [reply src/sections/Meshery/How-meshery-works/howitworks.style.js:74]: Screenshot 2024-12-25 at 11 44 14 AM

Comment (balagowda) [reply src/sections/Meshery/How-meshery-works/howitworks.style.js:74]: You can see the left div reaching the end.

Comment (balagowda) [reply src/sections/Meshery/How-meshery-works/howitworks.style.js:74]: And this is after adding margin
Screenshot 2024-12-25 at 11 48 31 AM

**Comment by l5io** [comment]
🚀 Preview for commit 1bbfd035e09a8d058b7d30ba5148853a06532545 at: https://676ba93eda0e8ed49751fc4a--layer5.netlify.app

**Review by sudhanshutech — APPROVED** [review]

**Pull request merged**